morse.rb

```diff
@@ -0,0 +1,71 @@
+
+    @dictionary = {
+    "a" => ".-",
+    "b" => "-...",
+    "c" => "-.-.",
+    "d" => "-..",
+    "e" => ".",
+    "f" => "..-.",
+    "g" => "--.",
+    "h" => "....",
+    "i" => "..",
+    "j" => ".---",
+    "k" => "-.-",
+    "l" => ".-..",
+    "m" => "--",
+    "n" => "-.",
+    "o" => "---",
+    "p" => ".--.",
+    "q" => "--.-",
+    "r" => ".-.",
+    "s" => "...",
+    "t" => "-",
+    "u" => "..-",
+    "v" => "...-",
+    "w" => ".--",
+    "x" => "-..-",
+    "y" => "-.--",
+    "z" => "--..",
+    "1" => ".----",
+    "2" => "..---",
+    "3" => "...--",
+    "4" => "....-",
+    "5" => ".....",
+    "6" => "-....",
+    "7" => "--...",
+    "8" => "---..",
+    "9" => "----.",  
+    "0" => "-----",
+    " " => " "
+    }
+
+def decode_char(string)
+    return (@dictionary.key(string)).upcase()
+end
+
+puts decode_char('--')
+
+def decode_word(str)
+  splitStr = str.split
+  word = ""
+  splitStr.each do |char|
+    word.concat(decode_char(char))
+  end
+  return word.upcase()
+end
+
+puts decode_word("-- -.--")
+
+def decode_message(message)
+  splitMessage = message.split("   ")
+  result = ""
+  splitMessage.each do |word|
+    result.concat(decode_word(word))
+    result.concat(" ") if word != splitMessage.last
+  end
+  return result.upcase()
+end
+
+puts decode_message("-- -.--   -. .- -- .")
+puts decode_message(".-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...")
```

Comment:
- [ ] The output we are getting here is different from what was expected from the project's requirement, would you fix this? 
### Your output
![Screenshot from 2022-08-17 18-30-18](https://user-images.githubusercontent.com/56886064/185193862-a8559321-e615-4d66-a0a3-6cf1091b17e7.png)

### Expected output
![Screenshot from 2022-08-17 18-32-15](https://user-images.githubusercontent.com/56886064/185193976-98d5e73e-e437-4b3d-87b7-fc5899677348.png)

Reply:
Thank you @descholar-ceo  for this kind and helpful feedback